packages/navstar/extra/dcos-navstar.service

```diff
@@ -21,4 +21,7 @@ ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-navstar
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-minuteman
 ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/navstar/mnesia
 ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/navstar/lashup
+ExecStartPre=/usr/bin/env modprobe dummy
+ExecStartPre=-/usr/bin/env ip link add minuteman type dummy
```

Comment:
+1 on @lingmann comment on converting this to script. We should just have one `ExecStartPre` point it to a shell and handle all failure conditions in the shell. Would help us debug errors a lot better.